src/typer.js

```diff
@@ -195,8 +195,8 @@
         var index = 0;
         var fn;
 
-        config.timer = $interval(function() {
-          element.html(word.substring(0, index + 1));
+        config.timer = $interval(function typerType() {
+          element[0].textContent = word.substring(0, index + 1);
```

Comment:
the `element` variable is the DOM element, `element[0]` is unnecessary. This will fix the failing test as it tries to call textContent on undefined.